.gitignore

```diff
@@ -39,5 +39,6 @@ CMakeLists.txt.user
 *.kdev4
 # Ignore IDEA/Clion files/dirs
 /.idea/
+Source/.idea
```

Comment:
I didn't notice this in the other large PR, but wouldn't it be a better idea (no pun intended) to adjust the first line to match all of them?

Because I doubt we'd ever have an `.idea` folder _anywhere_ that needs to go under version control.

Reply:
Sure, how should I adjust it?

Reply:
Good question. I _think_ `.idea/` should do the trick? `git status` will tell you after updating the file, no need to commit it first. Often takes me a few attempts until I get the folder syntax right 🤷

Reply:
It seems to work now